packages/cli/lib/DevServerManager.js

```diff
@@ -75,6 +80,11 @@ class DevServerManager {
             debug,
             onUploadRequired,
             promptUser,
+            logger,
```

Comment:
Just a thought, do we still need to pass in `debug` if we're also passing in the logger? I'm thinking the `debug` might be outdated by at this point since `logger.debug()` should do the same thing.

Reply:
Yeah, good call. I'll update this PR and the dev server to remove it